features/command_line/bisect.feature

```diff
@@ -54,12 +54,12 @@ Feature: Bisect
       Bisect started using options: "--seed 1234"
       Running suite to find failures... (0.16755 seconds)
       Starting bisect with 1 failing example and 9 non-failing examples.
+      Checking that failures are order-dependent... failure is order-dependent
 
-      Round 1: searching for 5 non-failing examples (of 9) to ignore: .. (0.30166 seconds)
-      Round 2: searching for 3 non-failing examples (of 5) to ignore: .. (0.30306 seconds)
-      Round 3: searching for 2 non-failing examples (of 3) to ignore: .. (0.33292 seconds)
-      Round 4: searching for 1 non-failing example (of 2) to ignore: . (0.16476 seconds)
-      Round 5: searching for 1 non-failing example (of 1) to ignore: . (0.15329 seconds)
+      Bisecting over non-failing examples 1-9 .. ignoring examples 6-9 (0.30166 seconds)
+      Bisecting over non-failing examples 1-5 .. ignoring examples 4-5 (0.30306 seconds)
+      Bisecting over non-failing examples 1-3 .. ignoring example 3 (0.33292 seconds)
+      Bisecting over non-failing examples 1-2 . ignoring example 1 (0.16476 seconds)
```

Comment:
I left it out because in this algorithm there aren't really sequential rounds (they're executed sequentially, but really the doubly-recursive case is a branch), so I couldn't think of a neat way to map the doubly-recursive case onto the old rounds-counting system.

I tried adding an object-level counter and incrementing it every time `bisect_over` is entered, but I didn't like the extra state (and it didn't really represent what the algorithm was doing very well). I'd be happy to put it back though if you think it's okay.